pandas/core/indexes/base.py

```diff
@@ -2530,20 +2539,20 @@ def dropna(self, how="any"):
     # --------------------------------------------------------------------
     # Uniqueness Methods
 
-    def unique(self, level=None):
+    def unique(self: _IndexT, level: Optional[Hashable] = None) -> _IndexT:
         """
         Return unique values in the index.
 
         Unique values are returned in order of appearance, this does NOT sort.
 
         Parameters
         ----------
-        level : int or str, optional, default None
+        level : int or hashable, optional
```

Comment:
"hashable" seems weird here.  may parenthetically "level name"?

Reply:
I've tried adding a text below the types.